coderd/coderdtest/coderdtest.go

```diff
@@ -421,6 +421,7 @@ func NewOptions(t testing.TB, options *Options) (func(http.Handler), context.Can
 			handler.ServeHTTP(w, r)
 		}
 	}))
+	t.Logf("coderdtest server listening on %s", srv.Listener.Addr().String())
```

Comment:
suggestion: extract for later re-use in `defer()` below

Reply:
I'm paranoid, and would prefer to re-query them.